community/imagemagick6/APKBUILD

```diff
@@ -3,7 +3,7 @@
 # Maintainer: Jakub Jirutka <jakub@jirutka.cz>
 pkgname=imagemagick6
 _pkgname=ImageMagick
-pkgver=6.9.9.47
+pkgver=6.9.9.51
```

Comment:
As of 21. Dec. 2018, `6.9.10-19` is the latest version, which successfully builds.

Reply:
6.9.10-x may require other packages to be rebuilt. I guess 6.9.9.51 is safe.